config/routes.rb

```diff
@@ -15,6 +15,8 @@
       only: [:create, :edit, :update]
   end
 
+  get 'my-account', to: 'users#show', as: 'my-account'
```

Comment:
I think I mentioned this earlier, but, Would you consider using an `index` action instead of a `show`?

I mean, what if we just retrieve data from the logged user instead of querying the user by the passed params.

If what I say makes sense to you, but you don't know how to start. I'd be more than happy to help you with it.

Reply:
In fact, actually is what you are doing XD

Reply:
I'd suggest to move it to an index, because `show` actions are supposed to receive an `id` via `url-params`